### DISCUSSION TITLE
How long does it take to get publisher verification?

### DISCUSSION THREAD

**Discussion by abdul-alhasany** [microsoft/vsmarketplace#710] [General]
Hi
I submitted my publisher details around a month ago (it has been a while, I can not remember exactly when). I still have not been verified. 

Is there a time estimate? Where/How can I follow up with this?

**Comment by mark-wiemer** [comment]
sad to see no response, I just submitted https://github.com/microsoft/vsmarketplace/discussions/995 as well
